packages/cli/lib/supportHyperlinks.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
```

Comment:
If we can, I think it's also worthwhile to look into removing `supports-color` as a dep and wrapping that functionality into this file (or maybe pull it into its own file). It would also be cool if we had some `getTerminalUISupport` util function that aggregated these color + hyperlinks utils and returned something along the lines of:
```js
{
 hyperlinks: true,
 color: false
}
```
Then all of the UI utils in [ui.js](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/ui.js) can use it to determine how to render output